app/components/authorCV/awardItem.tsx

```diff
@@ -48,18 +45,20 @@ class AwardItem extends React.PureComponent<AwardItemProps, AwardItemState> {
           id,
           title,
           receivedDate,
-          receivedDateYear: receivedDate.split("-")[0],
-          receivedDateMonth: receivedDate.split("-")[1],
+          relatedLink,
         }}
       />
     ) : (
       <div className={styles.itemWrapper}>
         <div className={styles.dateSectionWrapper}>
-          <span className={styles.dateContent}>{format(receivedDate, "MMM YYYY")}</span>
+          <span className={styles.dateContent}>{receivedDate}</span>
         </div>
         <div className={styles.contentWrapper}>
           {this.getEditItemButtons(id)}
           <span className={styles.awardTitleContent}>{title}</span>
+          <a className={styles.relatedLinkContent} href={!relatedLink ? "" : relatedLink}>
```

Comment:
When you use `conditional ternary operator`, please use a positive expression as much as possible for the condition.
In this case `relatedLink ? relatedLink : ""`